js/src/forum/addPostQuoteButton.js

```diff
@@ -32,6 +34,8 @@ export default function addPostQuoteButton() {
             const lastRect = rects[rects.length - 1];
             button.showEnd(lastRect.right, lastRect.bottom);
           }
+        } else {
+          button.hide();
```

Comment:
Isn't this already done by the onmousedown handler in the fragment itself?

Reply:
I believe so... But would we want to remove it in this PR?

Reply:
It was *added* in this PR.

Reply:
Lol I didn't notice that. Removed